VMEncryption/test/test_utils.py

```diff
@@ -8,6 +8,7 @@ def __init__(self, name, version, kernel):
         self.distro_info[1] = version
         self.kernel_version = kernel
         self.mount_path = 'mount'
+        self.mkdir_path = 'mkdir'
```

Comment:
Just a comment - As unit tests are moved under tox so we can test both python2 + python3 simultaneously, and with the goal of being able to run all tests within a container as part of build (not running in a pre-prepped Azure VM with a running guest agent or specific file system structure) - we'll need to mock everything and run as low privilege user.  Running mount and mkdir within the unit test itself push the boundaries of a unit test.  Not sure it has to be changed here, but is a a consideration that is going to be coming up moving forward.

Reply:
These are just dummy values. mount and mkdir will not actually be executed during unittest. It has be added as code expects these property to be present in DistroPatcher.
Sample here: https://github.com/Azure/azure-linux-extensions/blob/ade-singlepass-dev/VMEncryption/main/DiskUtil.py#L84
The actual execution of mount and mkdir is mocked in unittest.